sdks/python/apache_beam/typehints/intrinsic_one_ops_test.py

```diff
@@ -30,7 +30,7 @@ class IntrinsicOneOpsTest(unittest.TestCase):
   def testOperationsOrder(self):
     if sys.version_info >= (3, 12):
       dis_order = dis.__dict__['_intrinsic_1_descs']
-      beam_order = list()
+      beam_order = []
```

Comment:
nit:  use [list comprehensions](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions) for a more Pythonic syntax. e.g.

`beam_ops = [fn.__name_upper() for fn in intrinsic_one_ops.INT_ONE_OPS]`

Reply:
done